test/unit/printer/print_enums.cpp

```diff
@@ -0,0 +1,70 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Aina Niemetz, Gereon Kremer, Andres Noetzli
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2023 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Black box testing of printing enumerator types
+ */
+
+#include <cvc5/cvc5.h>
+
+#include <iostream>
+
+#include "test_smt.h"
+#include "theory/arith/nl/strategy.h"
+#include "theory/arith/rewrites.h"
+#include "theory/ext_theory.h"
+#include "theory/incomplete_id.h"
+#include "theory/inference_id.h"
+#include "theory/strings/rewrites.h"
+#include "theory/strings/strategy.h"
+
+using namespace cvc5::internal::theory;
+
+namespace cvc5::internal {
+namespace test {
+
+/**
+ * Set domain.optName to value due to reason. Notify if value changes.
+ */
+#define TEST_ENUM_RANGE(enumName, firstEnum, lastEnum) \
+  {                                                    \
```

Comment:
Are all these `\` necessary?

Reply:
Yes, macros in C++ require definitions like this.